sfaira/data/dataloaders/loaders/d10_1038_s41590_021_00933_1/human_blood_2021_citeseq_nathan_001.py

```diff
@@ -0,0 +1,20 @@
+import anndata
+import os
+import scipy.sparse
+import pandas as pd
+
+disease_map = {"CASE": "pulmonary tuberculosis", "CONTROL": "healthy"}
+
+
+def load(data_dir, sample_fn, **kwargs):
+    fn = os.path.join(data_dir, sample_fn)
+    adata = anndata.read_text(fn, delimiter="\t")
+    adata = adata.T
+    adata.X = scipy.sparse.csc_matrix(adata.X)
+    metadata = pd.read_csv(os.path.join(data_dir, "GSE158769_meta_data.txt.gz"), sep="\t")
+    adata.obs = adata.obs.join(metadata.set_index('cell_id'))
+    adata = adata[~adata.obs.cluster_name.isna()]
```

Comment:
Are these unlabeled cells that are putative doublets or bad quality cells? We could also label them as "unknow" ncells rather then filtering them out, this depends though on what their most likely identity is. This would imply adding a row to the cell type tsv which looks like this `na unknown unknown`. @Zethson could you check if this needs improving in the data contribution documentation?

Reply:
I checked percent mito and number of counts and they don't look like low quality cells and they also have UMAP coordinates. So changing them to "unknown" seems appropriate.

Reply:
Is this passing the data loader test now? It may because we chose the exact correct na cell type label in the loader . Even if it is already, let's add `na unknown unknown` (should work, otherwise `unknown unknown unknown` if you map to unknown in the loader) to the tsv so that we have a standardised place to update these descriptor fields in the future when we may update the unknown cell type identifiers, this is more difficult if they are defined in the data loader.

Reply:
It was passing the dataloader test already but I now added `na unknown unknown` to the tsv and removed mapping to unknown in the .py